tests/oscillating_collimator_let.py

```diff
@@ -83,3 +85,22 @@ def test_WHEN_input_values_cause_discriminant_to_be_negative_THEN_discriminant_p
 
         # Assert
         self.ca.assert_that_pv_is_number(DISCRIMINANT, 1.0)
+
+    def test_WHEN_laser_unstable_THEN_collimator_is_reported_moving(self):
+        # Act
+        # in normal operations the radius is not dynamic so set it first so it is considered in future calcs
+        # Set 0 first then 1 so buffer contents are definitively inconsistent
+        self.ca_mot.set_pv_value("DMC01:Galil0Bi5_STATUS", 0)
+        self.ca_mot.set_pv_value("DMC01:Galil0Bi5_STATUS", 1)
+
+        # Assert
+        self.ca.assert_that_pv_is("COLLIM_MOVING", "MOVING")
+    
+    def test_WHEN_laser_stable_THEN_collimator_is_reported_stopped(self):
+        # Act
+        # in normal operations the radius is not dynamic so set it first so it is considered in future calcs
+        # Will take ten seconds for buffer to have entirely consistent contents
+        self.ca_mot.set_pv_value("DMC01:Galil0Bi5_STATUS", 1, sleep_after_set=10)
+
+        # Assert
+        self.ca.assert_that_pv_is("COLLIM_MOVING", "STOPPED")
```

Comment:
Perhaps better to assert that the pv _was_ MOVING, and then becomes (and stays) STOPPED after about 10s? I can help you write this if needed.